package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gnip",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Comment:
What about version in `package.json`? They should match.

Reply:
This was bumped in my side because I use this as a stop gap until this pr is merged And bumped when npm version - 
When you do an npm version on your side you'll bump it.. or just edit the PR, if I do it in my side I'll break my existing code 
Another way around this would be that I would bump to version 2.2.5 -
What would you prefer?